### ISSUE TITLE
Crash when exit app after start and zoom in/zoom out map 

### ISSUE THREAD

**Issue by DucHM** [mapbox/mapbox-android-demo#48]
I'm run mapbox-android-demo and crash when exit app after start and zoom in/zoom out map with appear error following:

Unfortunately, Mapbox Dev Preview has stopped --> OK

The console logcat information:
com.mapbox.mapboxandroiddemo E/IMGSRV: kickresource.c:1130: Debug assertion failed!
com.mapbox.mapboxandroiddemo A/libc: Fatal signal 6 (SIGABRT) at 0x00001c2b (code=-6), thread 7211 (pboxandroiddemo)

My device using for test:
 - Model numer: ASUS_T00J
 - Android version: 4.4.2

Please support this issues. Any help will be appreciated.

**Comment by bleege** [comment]
Which version of the app are you using @DucHM?

**Comment by DucHM** [comment]
Thank Bleege,
I used version 3.2.0 as following:

dependencies {
    compile 'com.android.support:gridlayout-v7:23.1.1'
    compile 'com.android.support:appcompat-v7:23.1.1'
    compile ('com.mapbox.mapboxsdk:mapbox-android-sdk:3.2.0@aar'){
        transitive=true
    }
    compile('com.crashlytics.sdk.android:crashlytics:2.5.2@aar') {
        transitive = true;
    }
}

**Comment by DucHM** [comment]
I downloaded the demo from https://github.com/mapbox/mapbox-android-demo/ and run after changing the version of mapbox to 3.2.0 as above

**Comment by bleege** [comment]
@DucHM I just updated the App to run Mapbox Android 3.2.0 in #49.  I wasn't able to recreate the crash on zoom in / out as mentioned.  

It should be on Google Play in a few hours if you'd like to download it and run it from there to see if you're still encountering issues.  Also feel free to explore the [`master` branch here](https://github.com/mapbox/mapbox-android-demo/commits/master) to see what was changed.

As for the actual device, it looks like you're running an [Asus Zenfone 5](https://deviceatlas.com/device-data/devices/asus/t00j/6859536) correct?  I wasn't able to find much about what version of OpenGL it supports, but the Mapbox Android SDK requires OpenGL 2.0 and above.
